tests/test_redir.py

```diff
@@ -235,18 +231,23 @@ def test_run_with_console_redirect(console_settings, capfd):
 
         print("\n" * 1000)
         print("---------------")
+        time.sleep(1)
```

Comment:
why do you need to add this?

Reply:
these were in the tests before...I assumed it's due to potentially slow i/o on the CI to make sure all those tens of thousands of lines of rubbish are picked up.